Client/Frontend/Strings.swift

```diff
@@ -3772,20 +3772,20 @@ extension String {
             value: "Show products recommended by %@",
             comment: "Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.")
         public static let SettingsCardTurnOffButton = MZLocalizedString(
-            key: "Shopping.SettingsCard.TurnOff.Button.v120",
+            key: "Shopping.SettingsCard.TurnOffButton.Title.v120",
             tableName: "Shopping",
             value: "Turn Off Review Checker",
             comment: "Label of the button from settings card displayed in the shopping review quality bottom sheet.")
-        public static let SettingsCardExpandedAccessibilityLabel = MZLocalizedString(
+        public static let SettingsCardExpandAccessibilityLabel = MZLocalizedString(
             key: "Shopping.SettingsCard.Expanded.AccessibilityLabel.v120",
```

Comment:
The keys for this collapsible card need to be updated as well to match the value.

Reply:
good catch, updated